### PR TITLE
Default contacts demo to pinned AppBar

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/contacts_demo.dart [hunk 1]
```diff
@@ -93,7 +93,7 @@ class ContactsDemoState extends State<ContactsDemo> {
   static final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
   final double _appBarHeight = 256.0;
 
-  AppBarBehavior _appBarBehavior;
+  AppBarBehavior _appBarBehavior = AppBarBehavior.pinned;
```
Comment (abarth) [inline examples/flutter_gallery/lib/demo/contacts_demo.dart]: Why not default to AppBarBehavior.normal?

Comment (cbracken) [inline examples/flutter_gallery/lib/demo/contacts_demo.dart]: Mostly just consistency with the Gallery home (though I'll admit I like it better on iOS too). I can switch to normal.

Comment (cbracken) [inline examples/flutter_gallery/lib/demo/contacts_demo.dart]: On further poking at it, I don't think the behaviour for `normal` mode is right on iOS. I've filed #8464.

**Comment by abarth** [comment]
In any case, LGTM for whatever default you decide on.

**Pull request merged**